docs(graalvm): graalvm examples (#2499)
kestra-io/docs: content/docs/15.how-to-guides/python.md

```

## Execute GraalVM Task

Kestra also supports GraalVM integration, allowing you to execute Python code directly on the JVM, with the potential for performance improvements. There are currently two tasks:
- [Eval](/plugins/plugin-graalvm/python/io.kestra.plugin.graalvm.python.eval)
- [FileTransform](/plugins/plugin-graalvm/python/io.kestra.plugin.graalvm.python.filetransform)

> In this example, the `Eval` is used to manipulate data from a previous task. GraalVM makes it easy for us to generate outputs from variables in Python with the use of the `outputs` property. Useful if you want to manipulate data and pass the new format to another task.

Reviewer: ```suggestion
In this example, the `Eval` task is used to manipulate data from a previous task. GraalVM makes it easy to generate outputs from variables in Python using the `outputs` property. This is useful if you want to manipulate data and pass the new format to another task.
```
